darwin/importer/importer.py

```diff
@@ -267,6 +275,260 @@ def _resolve_annotation_classes(
+                )
+
+            # check if property and annotation class exists in team
+            if (a_prop.name, annotation_class_id) \
+                not in team_properties_annotation_lookup:
+
+                # check if fullproperty exists in create_properties
+                for full_property in create_properties:
+                    if full_property.name == a_prop.name and \
+                        full_property.annotation_class_id == annotation_class_id:
+                        # make sure property_values is not None
+                        if full_property.property_values is None:
+                            full_property.property_values = []
+
+                        property_values = full_property.property_values
+                        # find property value in m_prop (.v7/metadata.json) options
+                        for m_prop_option in m_prop_options:
+                            if m_prop_option.get("value") == a_prop.value:
+                                # update property_values with new value
+                                full_property.property_values.append(
+                                    PropertyValue(
+                                        position=m_prop_option.get("position"),  # type: ignore
+                                        type=m_prop_option.get("type"),  # type: ignore
+                                        value={"value": m_prop_option.get("value")},  # type: ignore
+                                        color=m_prop_option.get("color"),  # type: ignore
+                                    )
+                                )
+                                break
+                        break
+                else:
+                    property_values = []
+                    # find property value in m_prop (.v7/metadata.json) options
+                    for m_prop_option in m_prop_options:
+                        if m_prop_option.get("value") == a_prop.value:
+                            property_values.append(
+                                PropertyValue(
+                                    position=m_prop_option.get("position"),  # type: ignore
+                                    type=m_prop_option.get("type"),  # type: ignore
+                                    value={"value": m_prop_option.get("value")},  # type: ignore
+                                    color=m_prop_option.get("color"),  # type: ignore
+                                )
+                            )
+                            break
+                    # if it doesn't exist, create it
+                    full_property = FullProperty(
+                        name=a_prop.name,
+                        type=m_prop_type,  # type from .v7/metadata.json
+                        required=m_prop.required,  # required from .v7/metadata.json
+                        description="property-created-during-annotation-import",
```

Comment:
Don't we export description in the manifest? As far as I remember we do, and must take it from there instead of puting custom string

Reply:
description field is added in a new commit, and removed the type field too, also made sure the code dosen't break